templates/partial/_standard-navigation.html

```diff
@@ -0,0 +1,74 @@
+<header id="standard-navigation" class="p-navigation is-dark">
+  <div class="row">
+    <div class="p-navigation__banner col-3 u-no-padding--left">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95">
+          </div>
+          <span class="p-navigation__logo-title">Canonical</span>
+        </a>
+      </div>
+      <a href="#standard-navigation" class="p-navigation__toggle--open p-global-nav-titles" title="menu">Menu</a>
+      <a href="#standard-navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+
+    <nav class="p-navigation__nav col-9" aria-label="main-navigation">
+      <ul class="p-navigation__items js-show-nav" style="margin-left: -.75rem;">
+        <li class="p-navigation__item--dropdown-toggle" id="company-nav">
+          <a href="/company" aria-controls="company-menu" class="p-navigation__link">Company</a>
+          <ul class="p-navigation__dropdown" id="company-menu" aria-hidden="true">
+            <li>
+              <a href="/blog" class="p-navigation__dropdown-item">Blog</a>
+            </li>
+            <li>
+              <a href="/press-centre" class="p-navigation__dropdown-item">Press centre</a>
+            </li>
+            <li>
+              <a href="/documentation" class="p-navigation__dropdown-item">Documentation practice</a>
+            </li>
+            <li>
+              <a href="/projects" class="p-navigation__dropdown-item">Projects</a>
+            </li>
+          </ul>
+        </li>
+        <li class="p-navigation__item" id="products-nav">
+          <a href="/#products" class="p-navigation__link">Products</a>
+        </li>
+        <li class="p-navigation__item--dropdown-toggle" id="partners-nav">
+          <a href="/partners" aria-controls="partners-menu" class="p-navigation__link">Partners</a>
+          <ul class="p-navigation__dropdown" id="partners-menu" aria-hidden="true">
```

Comment:
and these ones too

Reply:
As mentioned, element is not showing that's why.